src/main/java/io/confluent/examples/streams/SessionWindowsExample.java

```diff
@@ -152,17 +158,21 @@ static KafkaStreams createStreams(final String bootstrapServers,
     playEventSerde.configure(serdeConfig, false);
 
     final StreamsBuilder builder = new StreamsBuilder();
-    builder.stream(Serdes.String(), playEventSerde, PLAY_EVENTS)
+    builder.stream(PLAY_EVENTS, Consumed.with(Serdes.String(), playEventSerde))
         // group by key so we can count by session windows
-        .groupByKey(Serdes.String(), playEventSerde)
-        // count play events per session
-        .count(SessionWindows.with(INACTIVITY_GAP), PLAY_EVENTS_PER_SESSION)
+        .groupByKey(Serialized.with(Serdes.String(), playEventSerde))
```

Comment:
indention of all lines should be 2

Reply:
not according to the confluent code style